src/Components/ImagesTable/ImageLink.js

```diff
@@ -9,13 +9,9 @@ import { useSelector } from 'react-redux';
 import ImageLinkDirect from './ImageLinkDirect';
 
 import { selectImageById } from '../../store/composesSlice';
-import { selectComposeById } from '../../store/composesSlice';
 
 const ProvisioningLink = ({ imageId, isExpired, isInClonesTable }) => {
   let image = useSelector((state) => selectImageById(state, imageId));
```

Comment:
Let's declare `image` as a `const` now that we no longer reassign it.

Reply:
sure! nice find :)
I wonder why linter is not bugging me for that, I'd expect this to be lintable.

Reply:
Because we don't have that rule in our eslintrc :wink: good point, I'll add it.